app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from __future__ import annotations
+
+from app.copy_content import manager_call
+
+
+def move_file(command: str) -> None:
+    split_cmd = command.split()
+    if len(split_cmd) != 3 or "mv" not in split_cmd:
```

Comment:
The condition `"mv" not in split_cmd` is not sufficient to validate the command. It should specifically check if `split_cmd[0]` is "mv" to ensure the command starts with the correct keyword.